backwardcompatibilityml/sweep_management.py

```diff
@@ -114,8 +118,12 @@ def get_sweep_status(self):
         return self.last_sweep_status
 
     def get_sweep_summary(self):
+        metric_name = self.performance_metric.__name__
+        if metric_name == "model_accuracy":
```

Comment:
There's no need to do this if check, because we always want to see the actual function name here.